### PR TITLE
Rename GL_INTERFACE to KODIGL_INTERFACE

### PR DESCRIPTION
**Pull request by fritsch**
Yeah. One commit a day keeps the trouble away

### CODE REVIEW

File: xbmc/windowing/X11/WinSystemX11GLContext.cpp [hunk 1]
```diff
@@ -268,7 +268,7 @@ bool CWinSystemX11GLContext::RefreshGLContext(bool force)
   std::transform(gpuvendor.begin(), gpuvendor.end(), gpuvendor.begin(), ::tolower);
   bool isNvidia = (gpuvendor.compare(0, 6, "nvidia") == 0);
   bool isIntel = (gpuvendor.compare(0, 5, "intel") == 0);
-  std::string gli = (getenv("GL_INTERFACE") != nullptr) ? getenv("GL_INTERFACE") : "";
+  std::string gli = (getenv("KODIGL_INTERFACE") != nullptr) ? getenv("KODIGL_INTERFACE") : "";
```
Comment (FernetMenta) [inline xbmc/windowing/X11/WinSystemX11GLContext.cpp]: I would rename all environment variables to KODI_, not KODIXX_

Why don't to search for all? It won't take you more than just a couple of minutes. If you don't do it now, it won't be done ever

**Comment by fritsch** [comment]
No problem. Will do tonight.

Rainer Hochecker <notifications@github.com> schrieb am Mo., 26. März 2018,
21:45:

**Comment by fritsch** [comment]
```
Application.cpp:    if (getenv("HOME"))
Application.cpp:      CSpecialProtocol::SetEnvHomePath(getenv("HOME"));
Application.cpp:  if (getenv("USER"))
Application.cpp:    userName = getenv("USER");
Application.cpp:  if (getenv("HOME"))
Application.cpp:    userHome = getenv("HOME");
Application.cpp:  if (getenv(envAppHome))
Application.cpp:    appPath = getenv(envAppHome);
Application.cpp:    if (getenv(envAppTemp))
Application.cpp:      strTempPath = getenv(envAppTemp);
Application.cpp:    if (getenv(envAppTemp))
Application.cpp:      strTempPath = getenv(envAppTemp);
Application.cpp:  if (getenv("USER"))
Application.cpp:    userName = getenv("USER");
Application.cpp:  if (getenv("HOME"))
Application.cpp:    userHome = getenv("HOME");
Util.cpp:  auto strPath = CEnvironment::getenv(strTarget);
addons/binary-addons/AddonDll.cpp:    std::string tempbin = getenv("XBMC_ANDROID_LIBS");
cores/DllLoader/DllLoaderContainer.cpp:  std::string systemLibs = getenv("XBMC_ANDROID_SYSTEM_LIBS");
cores/DllLoader/DllLoaderContainer.cpp:  std::string localLibs = getenv("XBMC_ANDROID_LIBS");
cores/DllLoader/Win32DllLoader.cpp:  { "getenv",                     -1, (void*)dll_getenv,                    NULL },
cores/DllLoader/exports/emu_msvcrt.cpp:  std::string apkPath = getenv("XBMC_ANDROID_APK");
cores/DllLoader/exports/emu_msvcrt.cpp:  char* dll_getenv(const char* szKey)
cores/DllLoader/exports/emu_msvcrt.h:  char* dll_getenv(const char* szKey);
cores/DllLoader/exports/msvcrt.def:      getenv
cores/DllLoader/exports/pncrt.def:      getenv
cores/DllLoader/ldt_keeper.c:  if (getenv("DYLD_BIND_AT_LAUNCH") == NULL)
filesystem/SMBFile.cpp:    std::string truehome(getenv("HOME"));
filesystem/SMBFile.cpp:    std::string home(getenv("HOME"));
interfaces/python/PythonInvoker.cpp:  char* dll_getenv(const char* szKey);
platform/Environment.cpp:std::string CEnvironment::getenv(const std::string &name)
platform/Environment.cpp:  return win_getenv(name);
platform/Environment.cpp:  char * str = ::getenv(name.c_str());
platform/Environment.h: * 	+ getenv
platform/Environment.h: * 	3. CEnvironment::getenv  returns a copy of environment variable value instead of pointer to value.
platform/Environment.h:   * \fn static std::string CEnvironment::getenv(const std::string &name);
platform/Environment.h:   * \sa xbmc_getenvUtf8, xbmc_getenvW
platform/Environment.h:  static std::string getenv(const std::string &name);
platform/Environment.h:  static std::string win_getenv(const std::string &name);
platform/android/activity/XBMCApp.cpp:  std::string apkPath = getenv("XBMC_ANDROID_APK");
platform/darwin/DarwinUtils.mm:    appHomeFolder = URIUtils::AddFileToFolder(getenv("HOME"), "Library/Application Support");
platform/darwin/osx/XBMCHelper.cpp:  m_launchAgentInstallFile = getenv("HOME");
platform/darwin/osx/XBMCHelper.cpp:  m_configFile = getenv("HOME");
platform/darwin/osx/XBMCHelper.cpp:  std::string strDir = getenv("HOME");
platform/posix/Filesystem.cpp:  auto result = getenv("TMPDIR");
platform/win10/Environment.cpp:  if (action == addOnly && !getenv(name).empty())
platform/win10/Environment.cpp:std::string CEnvironment::win_getenv(const std::string &name)
platform/win32/Environment.cpp:  if (action == addOnly && !getenv(name).empty() )
platform/win32/Environment.cpp:std::string CEnvironment::win_getenv(const std::string &name)
platform/win32/Environment.cpp:  wchar_t * wStr = ::_wgetenv(Wname.c_str());
storage/cdioSupport.cpp:    std::string strEnvDvd = CEnvironment::getenv("XBMC_DVD_DEVICE");
storage/linux/LinuxStorageProvider.cpp:  share.strPath = getenv("HOME");
storage/osx/DarwinStorageProvider.cpp:    share.strPath = getenv("HOME");
storage/osx/DarwinStorageProvider.cpp:  share.strPath = getenv("HOME");
utils/posix/SharedMemory.cpp:  char* xdgRuntimeDir = std::getenv("XDG_RUNTIME_DIR");
windowing/amlogic/WinSystemAmlogic.cpp:  const char *env_framebuffer = getenv("FRAMEBUFFER");
```

Should I rename the XBMC_ things too. I fear that there are some hidden dependencies from outside though.

**Comment by FernetMenta** [comment]
> Should I rename the XBMC_ things too

Yes, I would do it. Hidden dependencies are ugly and it's good if they come to the surface.

**Comment by wsnipex** [comment]
stuff like HOME and USER should be left as is

**Comment by pkerling** [comment]
The only ones that need changing from the list are those already prefixed with `XBMC_`

**Comment by fritsch** [comment]
Jep. Will do. Obviously clear not changing home :-)

pkerling <notifications@github.com> schrieb am Di., 27. März 2018, 15:19:

**Comment by Rechi** [comment]
@fritsch anything missing here?

**Comment by fritsch** [comment]
@Rechi I don't know -> most likely WIKI updates.

**Comment by Rechi** [comment]
So rebase and merge?

**Pull request merged**